README.md

```diff
@@ -28,18 +28,21 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
-tap "caskroom/fonts"
+tap "tap-from-non-standard-location", "https://user@bitbucket.org/user/homebrew-tap-repo.git"
+cask_args appdir: "/Applications"
+
+brew "denji/nginx/nginx-full", args: ["with-rmtp-module"]
```

Comment:
Put this one below the other examples so the first one is the simplest?

Reply:
I was going for alphabetical order, but sure.